src/table/types.ts

```diff
@@ -29,13 +32,14 @@ export interface ActionPayload {
   evt: React.KeyboardEvent | React.MouseEvent;
 }
 
+export type SelectionDispatch = React.Dispatch<TSelectionActions> | jest.Mock<any, any>;
```

Comment:
IMO test code should not leak into production code. I'd rather put a union type in a test/types file that adds the mock typing to `SelectionDispatch`. But I also saw that you just moved it, so never mind I guess. End rant :)

Reply:
haha yeah I see your point. I looked at that recently, not even sure we need to pass that mock to be able to test it. I will double-check. Not going to do anything about it in this PR though